.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+sudo: true
+branches:
+  only:
+    - master
+    - /^\d\.\d+$/
+    - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
+matrix:
```

Comment:
Is there a reason to use this mechanism rather than use tox env generation?

https://bitbucket.org/schinckel/django-boardinghouse/src/8a2320d4be97481bafec79d8f27f90d2638ac2d0/tox.ini?at=default&fileviewer=file-view-default

Reply:
Was in the template I was building from. We can change it.

Reply:
I am having trouble getting Travis to ignore the [tox] envlist otherwise, and travis is trying to build each env in each virtualenv.

Any suggestions?

Reply:
Hmm. I'm not sure. I don't generally use Travis (we use Codeship at work).

Reply:
Yeah, looking at https://github.com/funkybob/django-sniplates/blob/master/.travis.yml it appears that enumerating them is a solution...

Reply:
ok, I got it figured out. It needs tox-travis instead of tox.

Having said that, this is allowing me to build docs sep from 3.6.

https://travis-ci.org/jslay88/HAP-python/builds/343293928

vs. 

https://travis-ci.org/jslay88/HAP-python/builds/343290308

Reply:
The other thing that might be nice is coverage checking, but maybe we can do that later...

Reply:
I thought it would with `codecov`, but doesnt appear to be.